### PR TITLE
Bump curl to v7.86.0 to fix CVE-2022-42916.

### PR DESCRIPTION
**Pull request by dlorenc**
Signed-off-by: Dan Lorenc <dlorenc@chainguard.dev>

### CODE REVIEW

File: curl.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ package:
       attestation: TODO
       license: MIT
 
+secfixes:
```
Comment (kaniini) [inline curl.yaml:14]: 7.86.0-r0

File: melange.rsa [hunk 2]
```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (kaniini) [inline melange.rsa:1]: this shouldn’t be here, or the public key :)

**Review by kaniini — CHANGES_REQUESTED** [review]

File: curl.yaml (hunk 1 above)
Comment (dlorenc) [reply curl.yaml:14]: Fixed!

**Review by kaniini — APPROVED** [review]

File: melange.rsa (hunk 2 above)
Comment (luhring) [reply melange.rsa]: Maybe there's a setting or GH action we could use to check for this at some point 🤔  (e.g. gitleaks)

Comment (imjasonh) [reply melange.rsa]: We can add these paths to .gitignore, and grep for PRIVATE KEY in lint.sh

**Pull request merged**